packages/ember-data/lib/serializers/rest-serializer.js

```diff
@@ -529,12 +529,12 @@ var RESTSerializer = JSONSerializer.extend({
 
     In order for Ember Data to know that the model corresponding to
     the 'response-fast-car' hash is `FastCar` (modelName: 'fastCar'),
-    you can override typeForRoot to convert 'response-fast-car' to
+    you can override modelNameFromPayloadKey to convert 'response-fast-car' to
     'fastCar' like so:
```

Comment:
This line suggest returning `fastCar`, which is incorrect. Should only suggest returning dasherized names.